src/concrete/ml/quantization/quantized_module_passes.py

```diff
@@ -242,7 +243,7 @@ def integer_log2(value: float) -> Tuple[int, bool]:
             log2_value = int(numpy.rint(numpy.log2(value)))
             # Check that the integer power of two is close to the original value
             # with a small percentage tolerance
-            if numpy.isclose(numpy.power(2.0, log2_value), value, rtol=0.01):
+            if check_isclose_and_shape(numpy.power(2.0, log2_value), value, rtol=0.01):
```

Comment:
as said above, I think that this is a mistake originaly, we should use your new _allclose_ util instead here !

Reply:
Good catch